### ISSUE TITLE
[bug] Auto Complete won't work when typing in single line $$

### ISSUE THREAD

**Issue by WuYuanhun** [lukakerr/Pine#57]
Trying to use latex auto complete, it works when typing as plain text.
It works in multi-line $$.
But when I try to type between in single line $$,  the auto complete just does not pop out.

For example, auto complete works in:
```
$$
\A
$$
```
but does not work in :
```
$\A  $
```

**Comment by WuYuanhun** [comment]
Update the trigger conditions.
The auto complete only works when  the character beside '\' is in [ ' ', '\n', '\r'].

**Comment by WuYuanhun** [comment]
Start to wonder if this is a feature other than bug...

**Comment by lukakerr** [comment]
You’re right, there needs to be a whitespace character to the right otherwise autocomplete won’t trigger. But this should not occur inside inline and block maths, since it’s very common to be typing LaTeX commands inside $$ without whitespace.

**Comment by WuYuanhun** [comment]
I pull a merge #58 to fix it.
And test on my Mac successfully , (which may have little possibility to cause any problem because only changed a line of code)
Feel free to leave feedback
Thanks for review and Pine!
